### PR TITLE
YC-605 Dashboard filter object type options fix

### PR DESCRIPTION
**Pull request by danickfort**

### CODE REVIEW

**Review by rbovard — APPROVED** [review]
Thanks a lot for the improvement!

I've also added the same logic to filter `AdministrativeEntity` and `WorksType`, could you please review?

File: permits/filters.py [hunk 1]
```diff
@@ -56,6 +56,30 @@ class OwnPermitRequestFilterSet(BasePermitRequestFilterSet):
     pass
 
 
+def _get_administrative_entities_queryset_for_current_user(request):
+    return (
+        models.PermitAdministrativeEntity.objects.filter(
+            works_object_types__permit_requests__in=services.get_permit_requests_list_for_user(
```
Comment (danickfort) [inline permits/filters.py:62]: There is also a relation between `PermitAdministrativeEntity` and permit requests in via the `permit_requests` related name, so line 62 should be:
`permit_requests__in=services.get_permit_requests_list_for_user(`

**Review by danickfort — COMMENTED** [review]
Small change to have an optimized queryset @rbovard 🙂

**Comment by rbovard** [comment]
Thanks for the tip!

Please merge when tests are green.

**Pull request merged**